yoga/YGEnums.c

```diff
@@ -0,0 +1,234 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include "YGEnums.h"
+
+const char * YGStringifyAlign(const YGAlign value){
```

Comment:
Also `char *YGStringifyAlign` instead of `char * YGStringifyAlign` (asterix placement)